### PR TITLE
feat: show map modal before send post

### PR DESCRIPTION
**Pull request by WaDadidou**
### Created a new clean PR from >>> https://github.com/TERITORI/teritori-dapp/pull/898 <-- Don't hesitate to check this PR's history, issues and comments
___
This PR adds the Map modal and allows user to search for an address, using `react-native-leaflet-view` and `react-leaflet`.
Just front for now

image

### CODE REVIEW

File: packages/components/socialFeed/NewsFeed/MapComponent/AddressSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+import { BrandText } from "../../../BrandText";
+import { SVG } from "../../../SVG";
+import { TextInputCustom } from "../../../inputs/TextInputCustom";
+import { TextInputOutsideLabel } from "../../../inputs/TextInputOutsideLabel";
+
+import { useDebounce } from "@/hooks/useDebounce";
+
+interface NominatimSearchResult {
+  lat: string;
+  lon: string;
```
Comment (clegirar) [inline packages/components/socialFeed/NewsFeed/MapComponent/AddressSearch.tsx:28]: Maybe name that `lng` to fit with `LatLng` types names ?

Comment (clegirar) [reply packages/components/socialFeed/NewsFeed/MapComponent/AddressSearch.tsx:28]: Else LGTM 👍

**Comment by n0izn0iz** [comment]
I cleaned a bit the address search component in https://github.com/TERITORI/teritori-dapp/pull/1144/commits/66cc39a624c6b18b27e06e62f7632c409596aaf2

Please see https://github.com/TERITORI/teritori-dapp/blob/main/CONTRIBUTING.md#handling-client-provided-data

**Review by clegirar — APPROVED** [review]
LGTM 👍

**Pull request merged**